WordSearch.java

```diff
@@ -0,0 +1,61 @@
+// 79.
+//time - O(m * n)
```

Comment:
It exponential. Time Complexity: O(N⋅4^L ) where NN is the number of cells in the board and LL is the length of the word to be matched